app/assets/stylesheets/components.scss

```diff
@@ -0,0 +1,19 @@
+// Place all the styles related to the components controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.app-certificate-tag-expiring .govuk-tag {
+  background-color: #CB4B0B;
```

Comment:
that colour is not on the Design System palette: https://design-system.service.gov.uk/styles/colour/ @christopherthomasdesign

Reply:
The orange on the palette in the design system doesn't have enough contrast with white text so I darkened it to be WCAG AA compliant. Would be just as happy using the `govuk-error-colour` red but see your point about it maybe being too alarming... Not sure black text on a yellow background would be better in that respect either. I think we have a good enough reason to add a new colour here.